packages/api/src/utility.ts

```diff
@@ -65,20 +65,20 @@ export function mapSubscriptionsAsCsv(
     csvStr +=
       [
         user?.id,
-        `${user?.firstName ?? ''}`,
-        `${user?.name ?? ''}`,
-        `${user?.preferredName ?? ''}`,
+        `${sanitizeCsvContent(user?.firstName)}`,
```

Comment:
@michael-scheurer There is no reason to wrap it in backticks/interpolation here, right? Would remove for readability.